credoai/lens.py

```diff
@@ -191,49 +176,58 @@ def create_reports(self, report_name, export=False, display_results=False):
                     reporter.plot_results()
             else:
                 logging.info(f"No reporter found for assessment-{name}")
-        final_report = MainReport(report_name, reporters.values())
+        final_report = MainReport(f"{ci.RISK.title()} Report", reporters.values())
         final_report.create_report(self)
         # exporting
-        names = self.get_artifact_names()
-        name_for_save = f"{report_name}_model-{names['model']}_data-{names['dataset']}.html"
-        if isinstance(export, str):
-            final_report.write_notebook(
-                path.join(export, name_for_save), as_html=True)
-        elif export:
+        if export:
+                self.export_asessments(export, report=final_report)
```

Comment:
It seems `export_asessments` must be `export_assessments`
@IanAtCredo

Reply:
fixed